app/models/manageiq/providers/microsoft/infra_manager/refresh_parser.rb

```diff
@@ -314,18 +314,21 @@ def build_dvd_hash(dvd)
     end
 
     def process_vm_hardware(vm)
-      p = vm[:Properties][:Props]
+      p    = vm[:Properties][:Props]
+      cpus = p[:CPUCount]
 
       {
-        :cpu_total_cores    => p[:CPUCount],
-        :guest_os           => p[:OperatingSystem][:Props][:Name],
-        :guest_os_full_name => p[:OperatingSystem][:Props][:Name],
-        :memory_mb          => normalize_blank_property_num(p[:Memory]),
-        :cpu_type           => normalize_blank_property_str(p[:CPUType]),
-        :disks              => process_disks(p),
-        :networks           => process_hostname_and_ip(vm),
-        :guest_devices      => process_vm_guest_devices(vm),
-        :bios               => p[:BiosGuid]
+        :cpu_sockets          => cpus,
+        :cpu_cores_per_socket => 1,
```

Comment:
At first I was worried about having `cpu_cores_per_socket` as always `1`, but looking at some articles online it looks like this is really the case.  SCVMM doesn't surface this level of CPU details to the VMs.  It only surfaces the number of vCPUs associated to a VM.